playbooks/synchronise_ssh_keys.yml

```diff
@@ -0,0 +1,21 @@
+---
+- hosts: all
+  tasks:
+    - name: Clone the credential repo
+      git:
+        repo: '{{ credential_git_repo }}'
+        version: '{{ credential_git_ref }}'
+        dest: /tmp/credential_repo
+    - name: Add the authorised SSH public keys to all hosts.
+      authorized_key:
+        user: root
```

Comment:
Would be good if this could be configurable. There might be scenarios where we need to set it to something other than root

Reply:
good idea